internal/model/export_test.go

```diff
@@ -0,0 +1,4 @@
+package model
+
+// NewImports exported for tests.
+var NewImports = newImports
```

Comment:
This is very personal so you can disregard it but I find such exports very weird.
You are declaring a test file but are maintaining the main package which is strange to begin with.
Then you are exposing something internal.

I personally would make the constructors public and just use those. Your package is eitherway internal so can't be used by anyone outside. And the models are encapsulated enough that if anyone wants to use them for some reason, then let them do so.

Reply:
P.S. You are already using all other types that happen to have their fields public.

Reply:
I'm trying to use the pattern sparingly, but I do not dislike it in general.

I've seen this pattern across the standard library and find it useful when I don't want to expose a method or variable to the outside world, but it's hard to test a functionality because of that decision.

With that said - this source residing under /internal/ it may be clearer to just export the NewImports indeed.